anesthetic/plot.py

```diff
@@ -304,23 +303,34 @@ def hist_1d(ax, data, *args, **kwargs):
 
     xmin = kwargs.pop('xmin', None)
     xmax = kwargs.pop('xmax', None)
+    plotter = kwargs.pop('plotter', '')
+    weights = kwargs.pop('weights', None)
     if xmin is None:
-        xmin = data.min()
+        xmin = quantile(data, 0.01, weights)
     if xmax is None:
-        xmax = data.max()
+        xmax = quantile(data, 0.99, weights)
     histtype = kwargs.pop('histtype', 'bar')
 
-    plt.sca(ax=ax)
-    h, edges, bars = hist(data, range=(xmin, xmax), histtype=histtype,
-                          *args, **kwargs)
-    # As the y-axis on the diagonal 1D plots of the triangle plot won't
-    # be labelled, we set the maximum bar height to 1:
+    if plotter == 'astropyhist':
+        try:
+            print(xmin, xmax)
+            print(data)
+            print(weights)
+            h, edges, bars = hist(data, ax=ax, range=(xmin, xmax),
```

Comment:
A bit late, but I just noticed this now. We don't need those print statements any longer, right? 

Should this be addressed in a new PR?

Reply:
Opened #51